application/gui/properties_window.py

```diff
@@ -193,6 +197,8 @@ def _rename_item(self, widget=None, data=None):
 
 	def _create_monitor(self):
 		"""Create item monitor"""
+		if hasattr(self, '_monitor') and self._monitor is not None and not self._monitor.is_cancelled():
+			self._monitor.cancel()
```

Comment:
Why is this being done like this? Monitors should be automatically canceled on path cancel.